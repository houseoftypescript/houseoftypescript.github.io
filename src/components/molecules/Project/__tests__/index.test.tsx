import { render } from '@testing-library/react';
import { Project } from '../index';
import { StaticImageData } from 'next/image';

describe('Project', () => {
  it('renders correctly', () => {
    const button = render(<Project image={{} as StaticImageData}></Project>);
    expect(button.container).toMatchSnapshot();
  });
});
