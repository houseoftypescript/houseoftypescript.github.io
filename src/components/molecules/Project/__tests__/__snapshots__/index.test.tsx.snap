// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Project renders correctly 1`] = `
<div>
  <div
    class="hover:p-4 hover:shadow hover:border transition-all duration-100"
  >
    <a
      class="block"
      href=""
      target="_blank"
    >
      <div
        class="aspect-video rounded border bg-cover bg-center"
        style="background-image: url(undefined);"
      />
      <h2
        class="text-lg font-bold mt-4"
      />
    </a>
    <p
      class="text-gray-500 mt-4"
    />
    <div
      class="mt-4"
    />
  </div>
</div>
`;
